README.md

```diff
@@ -35,7 +35,7 @@ The following table shows the compatibilities of different versions of the bundl
 
 | FOSElasticaBundle                                                                       | Elastica | Elasticsearch | Symfony    | PHP   |
 | --------------------------------------------------------------------------------------- | ---------| ------------- | ---------- | ----- |
-| [6.0] (master)                                                                          | ^7.1     | 7.\*          | ^4.4\|^5.3 | >=7.4 |
+| [6.0] (master)                                                                          | ^7.1     | 7.\*          | ^4.4\|^5.3|^6.0 | >=7.4 |
```

Comment:
You should escape the `|` character inside a table cell

Reply:
Oh yeah missed it, thanks fixed it !